src/includes/class-alma-wc-admin-form.php

```diff
@@ -539,18 +594,106 @@ private function get_custom_fields_payment_method( $payment_method_name, $title,
 			'title_' . $payment_method_name       => array(
 				'title'       => __( 'Title', 'alma-woocommerce-gateway' ),
 				'type'        => 'text',
+				'class'       => 'i18n',
 				'description' => __( 'This controls the payment method name which the user sees during checkout.', 'alma-woocommerce-gateway' ),
 				'default'     => $default_settings[ "title_$payment_method_name" ],
 				'desc_tip'    => true,
 			),
 			'description_' . $payment_method_name => array(
 				'title'       => __( 'Description', 'alma-woocommerce-gateway' ),
 				'type'        => 'text',
+				'class'       => 'i18n',
 				'desc_tip'    => true,
 				'description' => __( 'This controls the payment method description which the user sees during checkout.', 'alma-woocommerce-gateway' ),
 				'default'     => $default_settings[ "description_$payment_method_name" ],
 			),
 		);
 	}
 
+
+	/**
+	 * For admin url : /wp-admin/admin.php?page=wc-settings&tab=checkout&section=alma
+	 *
+	 * @return void
+	 */
+	public function admin_head() {
```

Comment:
this class is a helper to build a form

it is not supposed to manage HTTP request.

please create a class to do this work (you can call this class `Alma_WC_Admin_Internationalization_Front_Helper` for example) 

this is a different class of `Alma_WC_Internationalization` because the Admin Front Helper deal with `$_GET` data where `Alma_WC_Internationalization` is just helping us to translate some strings and check wpml compatibility